example/src/main/java/io/netty/example/util/ServerUtil.java

```diff
@@ -0,0 +1,35 @@
+package io.netty.example.util;
+
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.ssl.util.SelfSignedCertificate;
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+public class ServerUtil {
+
+    private static final InternalLogger logger = InternalLoggerFactory.getInstance(ServerUtil.class);
+
+    private ServerUtil() {
+    }
+
+    static final boolean SSL = System.getProperty("ssl") != null;
+
+    public static SslContext getSslContext() {
```

Comment:
I think we should rename this to `buildServerSslContext`

Reply:
Thanks for your suggestion.